Automattic/wp-calypso
# Update/package whats new wp data

#### Changes proposed in this Pull Request
* This PR moves implements Redux state management for the What's New component.
* Migrates the ETK version of What's New to the package component version.

#### Testing instructions

* Checkout this PR and start Calypso.
* Sync ETK to your sandbox.
* Sandbox a test site.
* Load the post editor using the sandboxed and synced version of ETK.
* navigate to the MC What's New admin tool and add a new announcement.
* Wait for the endpoint cache to update (max of 10 minutes)
* In the editor sidebar, click What's New
* Confirm that the Announcements from the MC tool are displayed.

Related to #
* 50224
* D57620-code
